lib/tasks/audits.rake

```diff
@@ -1,24 +1,35 @@
 # TRANSLATORS: do not translate
 desc <<-END_DESC
-Expire old audits automatically
+Expire or anonymize old audits automatically
 
 Available conditions:
   * days        => number of days to keep audits (defaults to 90)
 
   Example:
     rake audits:expire # expires all audits older then 90 days
     rake audits:expire days=7 # expires all audits older then 7 days
+    rake audits:anonymize days=7 # anonymizes all audits older then 7 days
 
 END_DESC
 
 namespace :audits do
   task :expire => :environment do
-    before = ENV['days'].to_i.days.ago if ENV['days']
-    before ||= 90.days.ago
-    audits = Audited::Audit.up_until(before)
-    puts "Deleting audits older than #{before}. This might take a few minutes..."
+    audits = get_audits
+    puts "Deleting old audits. This might take a few minutes..."
```

Comment:
I think it would be good to keep the date in message, how about extracting `before = ENV['days'].to_i.days.ago if ENV['days']` to a method with memoization, something like


```ruby
def before_date
  @before_date ||= ENV['days'].to_i.days.ago || 90.days.ago
end
```

that way you can still use it in this message and perhaps even in anonymize task